src/mcp/prompts/core/init.ts

```diff
@@ -18,14 +19,38 @@ export const init = prompt(
     },
   },
   async ({ prompt }, { config, projectId, accountEmail }) => {
+    const shortCircuitUrls: Record<string, string> = {
+      "ai-logic": "firebase://guides/init/ai",
+      backend: "firebase://guides/init/backend",
+    };
```

Comment:
Let's move this out of the prompt fn definition and have it be something like `const GUIDE_PARAMS = {...}` then check against the constant object in this spot..

Reply:
Done